src/kirby/components/list/list.component.scss

```diff
@@ -79,6 +79,32 @@ $shadow: get-elevation(2);
   }
 }
 
+:host(.no-shape) {
```

Comment:
Vi kunne vel egentlig bruge attribute selector her? Var klasser ikke en nødvendighed fra {N} som ikke understøttede `:host` selectoren.

Reply:
Stærkt! I så fald bør vi vel lave det om for shape også?

Reply:
@jonasjensen-brolstark Nope, kun hvis attributen er "statisk" defineret dvs. `<my-tag my-attribute="hest" ...`
Hvis der bliver bundet til den, så konverterer Angular den til noget andet: . `<my-tag [my-attribute]="'hest'" ...` => <my-tag ng-reflect-my-attribute="hest" ...

Reply:
Modtaget

Reply:
🤦‍♂ Friday afternoon review. Disregard my comments.

I also forgot i write in english.